### DISCUSSION TITLE
Update to Chinese (Simplified) locale [zh_CN]

### DISCUSSION THREAD

**Discussion by bio1020** [aclap-dev/video-downloadhelper#635] [Language & Translation]
Translation for Video DownloadHelper Version 9

[zh_CN.txt](https://github.com/aclap-dev/video-downloadhelper/files/15442560/zh_CN.txt)

**Comment by paulrouget** [comment]
Thank you! Hope to get that in version 9.0.1.25 ❤️

**Comment by sexgg** [comment]
Chinese locale for VDHelper v9.0.2.3
[zh_CN.txt](https://github.com/user-attachments/files/15813241/zh_CN.txt)
